src/features/regexp-v-flag.md

```diff
@@ -0,0 +1,252 @@
+
+The latest example of this is [the new `unicodeSets` mode, enabled using the `v` flag](https://github.com/tc39/proposal-regexp-v-flag). This new mode unlocks support for _extended character classes_, including the following features:
+
+- [Unicode properties of strings](/features/regexp-v-flag#unicode-properties-of-strings)
+- [set notation + string literal syntax](/features/regexp-v-flag#set-notation)
+- [improved case-insensitive matching](/features/regexp-v-flag#ignoreCase)
+
+This article dives into each of these. But first things first — here’s how to use the new flag:
+
+```js
+const re = /…/v;
+```
+
+The `v` flag can be combined with existing regular expression flags, with one notable exception. The `v` flag enables all the good parts of the `u` flag, but with additional features and improvements. Crucially, `v` is a completely separate mode from `u` rather than a complementary one. For this reason, the `v` and `u` flags cannot be combined — trying to use both flags on the same regular expression results in an error. The only valid options are: either use `u`, or use `v`, or use neither `u` nor `v`. But since `v` is the most feature-complete option, that choice is easily made…
+
+Let’s dig into the new functionality!
+
+## Unicode properties of strings
+
+The Unicode Standard assigns various properties and property values to every symbol. For example, to get the set of symbols that are used in the Greek script, search the Unicode database for symbols whose `Script_Extensions` property value includes `Greek`.
+
+ES2018 Unicode character property escapes make it possible to access these Unicode character properties natively in ECMAScript regular expressions. For example, the pattern `\p{Script_Extensions=Greek}` matches every symbol that is used in the Greek script:
+
+```js
+const regexGreekSymbol = /\p{Script_Extensions=Greek}/u;
+regexGreekSymbol.test('π');
+// → true
+```
+
+By definition, Unicode character properties expand to a set of code points, and can thus be transpiled as a character class containing the code points they match individually. For example, `\p{ASCII_Hex_Digit}` is equivalent to `[0-9A-Fa-f]`: it only ever matches a single Unicode character/code point at a time. In some situations, this is insufficient:
+
+```js
+// Unicode defines a character property named “Emoji”.
+const re = /^\p{Emoji}$/u;
+
+// Match an emoji that consists of just 1 code point:
+re.test('⚽'); // '\u26BD'
+// → true ✅
+
+// Match an emoji that consists of multiple code points:
+re.test('👨🏾‍⚕️'); // '\u{1F468}\u{1F3FE}\u200D\u2695\uFE0F'
+// → false ❌
+```
+
+In the above example, the regular expression doesn’t match the 👨🏾‍⚕️ emoji because it happens to consist of multiple code points, and `Emoji` is a Unicode _character_ property.
```

Comment:
Side comment on "character": is "Unicode character property" the what Unicode calls it officially? Between concepts of "code unit" and "code point", I thought calling stuff "characters" was discouraged.

Reply:
https://unicode.org/reports/tr18/#domain_of_properties

> Typically the important information is for the subset of the code points that are characters; therefore, those properties are often also called properties of characters.
> 
> In addition to properties of characters, there are also properties of strings (sequences of characters). A property of strings is more general than a property of characters. In other words, any property of characters is also a property of strings; its domain is, however, limited to strings consisting of a single character.

Reply:
Perhaps @markusicu @macchiati have some guidance here?

Reply:
The quotation shows the reason.